Ginger/Ginger/User/UserProfile.cs

```diff
@@ -278,6 +279,9 @@ public void AddSolutionToRecent(Solution loadedSolution)
         [IsSerializedForLocalRepository]
         public string SolutionSourceControlProxyPort { get; set; }
 
```

Comment:
Give default value here using attribute like we did with boolean fields recently